### ISSUE TITLE
Trying to build master HEAD a0001c473 fails

### ISSUE THREAD

**Issue by tylert** [mitchellh/packer#3173]
Attempting to checkout commit a0001c473c532650776a835bd40968c9bdd7e162 and build it results in a test failure when trying to run "make":

```
2016/02/09 13:55:19 SSH proxy: serving on test
2016/02/09 13:55:19 Accept() called
2016/02/09 13:55:19 listen.Accept failed: accept error
2016/02/09 13:55:19 Accept() called
2016/02/09 13:55:19 Accept() called
2016/02/09 13:55:19 SSH proxy: accepted connection
2016/02/09 13:55:19 failed to handshake
2016/02/09 13:55:19 listen.Accept failed: accept error
2016/02/09 13:55:19 Accept() called
2016/02/09 13:55:19 done, serving an error
2016/02/09 13:55:19 8c4b26fb8635edbb1662 does not exist
--- FAIL: TestProvisionerPrepare_AuthorizedKeyFile (0.00s)
	provisioner_test.go:165: should have error
FAIL
FAIL	github.com/mitchellh/packer/provisioner/ansible	0.056s
```

If you comment out line 165 of provisioner/ansible/provisioner_test.go, everything else continues and works correctly (and all my existing templates function properly).

I emptied out my GOPATH and tried it again in order to confirm there wasn't a problem with the vendor stuff.

Currently building/running with:  VirtualBox 5.0.14, Mac OS X El Capitan 10.11.3, go 1.5.3.

**Comment by cbednarski** [comment]
Related to https://github.com/mitchellh/packer/issues/3095

**Comment by tylert** [comment]
Awesome!  Lemmie know if there's something you require me to try out.

**Comment by cbednarski** [comment]
This was fixed in #3159
